Formula/neal.rb

```diff
@@ -0,0 +1,66 @@
+class Neal < Formula
+  desc "Analyzes source code based on user-specified rules written in a custom Domain Specific Language."
+  homepage "https://uber.github.io/NEAL/"
+  url "https://github.com/uber/NEAL/archive/v0.2.2.tar.gz"
+  sha256 "084e4de62ec027d16fa965727764d801b9ba7939fb6205705993ab36585c328c"
+
+  depends_on "camlp4" => :build
+  depends_on "ocaml" => :build
+  depends_on "ocamlbuild" => :build
+  depends_on "opam" => :build
+
+  def install
+    ENV.deparallelize
+
+    opamroot = buildpath/"opamroot"
+    opamroot.mkpath
+    ENV["OPAMROOT"] = opamroot
+
+    ENV["OPAMYES"] = "1"
+    ENV["NATIVE"] = "1"
+    ENV["LIB_PATH"] = lib
+
+    # otherwise we get `undefined symbol _clock_time` in El Capitan due to Core
+    ENV.delete("SDKROOT") if MacOS.version < :sierra
```

Comment:
This needs to be reported upstream

Reply:
Sorry for the ignorance, but when you say upstream, do you mean the NEAL repo? Or NEAL's dependency?

Reply:
we no longer set SDKROOT so this line should be able to just be removed